### DISCUSSION TITLE
Access the user who moves the camera, using PTZ controls

### DISCUSSION THREAD

**Discussion by m3hdiIsaloo** [AxisCommunications/discussions#258] [Axis Windows SDKs] [answered]
I wrote a software, that controls ptz cameras for specific needs. I accessed the PTZ status using the Media Parser SDK, and now an event in the program, fires when someone changes PTZ status. One thing that I couldn't find in the documentation, is "who changes the PTZ status". if someone is logged in to use PTZ controls, I want to access the username. thanks for your support

**Comment by vivekatoffice** [comment, marked as answer]
Hi @m3hdiIsaloo ,
Really a good use case.
Have you had a chance to explore potential solutions or make any progress? Your insights would be greatly appreciated.

My Input:
If any how we can find the user calling the API, you can use [Dynamic text overlay](https://www.axis.com/vapix-library/subjects/t10175981/section/t10007638/display?section=t10007638-t10003585) to update the overlay.

![image](https://github.com/AxisCommunications/discussions/assets/52032125/96e07c49-760e-4a62-ae7e-45afc7f5a99e)
